src/css.js

```diff
@@ -117,43 +117,29 @@ function augmentWidthOrHeight( elem, name, extra, isBorderBox, styles ) {
 
 function getWidthOrHeight( elem, name, extra ) {
 
-	// Start with offset property, which is equivalent to the border-box value
-	var val,
-		valueIsBorderBox = true,
+	// Start with computed style
+	var valueIsBorderBox,
 		styles = getStyles( elem ),
+		val = curCSS( elem, name, styles ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
-	// Support: IE <=11 only
-	// Running getBoundingClientRect on a disconnected node
-	// in IE throws an error.
-	if ( elem.getClientRects().length ) {
-		val = elem.getBoundingClientRect()[ name ];
+	// Fall back to uncomputed css if necessary
+	if ( val < 0 || val == null ) {
```

Comment:
> The condition val <= 0 was made more strict: val < 0.

That was a different condition. The diff might make it look that way. If you look closer, you'll notice the condition with `val <= 0` was completely removed because it was checking the result from `getBoundingClientRect`, which was also removed. The `val < 0` condition was already there checking the result from `getCSS`, which we now do first.

> By changing what's initially tried from getBoundingClientRect to getCSS you eliminated the possibility of the value not being defined.

This was essentially true before. Again, this condition was already there after `getCSS`.